net/upnp/src/www/services_upnp.php

```diff
@@ -382,6 +391,15 @@ function miniupnpd_validate_port($port)
                     </tr>
                   </thead>
                   <tbody>
+                    <tr>
+                      <td><a id="help_for_num_permuser" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Number of permissions");?></td>
+                      <td>
+                        <input name="num_permuser" type="number" min="1" value="<?=$pconfig['num_permuser'];?>" />
```

Comment:
we do not use min= in our code as far as I know. To avoid side effects it would be better to remove it and let the validation handle that properly like it does now?

Reply:
Fair enough.  I can take out the `min` parameter.